features/user/types/User.ts

```diff
@@ -0,0 +1,19 @@
+import { Invitation } from './Invitation'
+
+export type User = {
+	id: string
+	name: string | null
+	image: string | null
+	email: string | null
+	password: string | null
```

Comment:
nie lepiej name?:string ? Chyba że masz tak zdefiniowane w jakiejś priśmie że zwraca string | null to w takim razie niech tak zostaną